packages/genui/a2ui-playground/src/pages/PlaybackPage.css

```
@@ -0,0 +1,201 @@
+  border: 1px solid rgba(0, 112, 243, 0.2);
+}
+
+.playbackChunkJson {
+  margin: 0;
+  padding: 8px 10px;
+  font-family: var(--geist-mono);
+  font-size: 12px;
+  line-height: 1.55;
+  color: var(--geist-secondary);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)